test/cmd/volumes.sh

```diff
@@ -29,6 +29,7 @@ os::cmd::expect_success 'oc volume dc/test-deployment-config --add --name=vol0 -
 os::cmd::expect_success 'oc set volume dc/test-deployment-config --add --name=vol2 --type=emptydir -m /opt'
 os::cmd::expect_failure_and_text "oc set volume dc/test-deployment-config --add --name=vol1 --type=secret --secret-name='\$ecret' -m /data" 'overwrite to replace'
 os::cmd::expect_success "oc set volume dc/test-deployment-config --add --name=vol10 --secret-name='my-secret' -m /data-2"
+os::cmd::expect_success "oc set volume dc/test-deployment-config --add --name=vol11 --configmap-name='my-configmap' -m /data-21"
```

Comment:
Don't we want to probe the resulting object with a `jsonpath` to ensure that what we did worked? For all of the tests in that vein, as well. I don't really see the current test as being that useful, `os.Exit(0)` would pass it.